index.js

```diff
@@ -6,7 +6,12 @@ module.exports = class PostgresDatabase extends HanaDatabase {
   constructor(...args) {
     super(...args)
     // maybe do more stuff
-    this._pool = new Pool(this.options.credentials)
+    // Cloud Foundry provides the user in the field username the pg npm module expects user
+    var credentials = this.options.credentials
+    if (credentials && credentials.username) {
```

Comment:
`this.options.credentials` have to be defined here anyways in order to make the connect work
so why not do 
```js
this.options.credentials.user = this.options.credentials.username ? this.options.credentials.username : this.options.credentials.user
this._pool = new Pool(this.options.credentials)
```

Reply:
Please check the new commit.